plugins/rest/resturl/rest_url.go

```diff
@@ -66,4 +92,39 @@ const (
 const (
 	// Routes is rest static route path
 	Routes = "/vpp/dump/v1/routes"
+	// Arps is rest ARPs path
+	Arps = "/vpp/dump/v1/arps"
+	// ProxyArps is rest proxy ARPs path
+	ProxyArps = "/vpp/dump/v1/proxyarp"
+	// PArpIfs is rest proxy ARP interfaces path
+	PArpIfs = "/vpp/dump/v1/proxyarp/interfaces"
+	// PArpRngs is rest proxy ARP ranges path
+	PArpRngs = "/vpp/dump/v1/proxyarp/ranges"
+)
+
+// L4 plugin
+const (
+	// Sessions is rest L4 configuration path
+	Sessions = "/vpp/dump/v1/sessions"
+)
+
+// Command
+const (
+	// Command allows to put CLI command to the rest
+	Command = "/vpp/command"
+)
+
+// Telemetry
+const (
+	// Telemetry allows to put CLI
+	Telemetry  = "/vpp/dump/v1/telemetry"
```

Comment:
I think that `dump/v1` part is not very fitting for telemetry. We should just keep that for things that are actually dump via binapi and that contain the NB models.